lib/ui/forgot_password/forgot_password_screen.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:survey_flutter_ic/gen/assets.gen.dart';
+import 'package:survey_flutter_ic/ui/widget/input_field_widget.dart';
+import 'package:survey_flutter_ic/utils/dimension.dart';
+
+class ForgotPasswordScreen extends StatelessWidget {
+  const ForgotPasswordScreen({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        backgroundColor: Colors.transparent,
+        body: Container(
+          decoration: BoxDecoration(
+            image: DecorationImage(
+              image: AssetImage(Assets.images.bgLoginOverlay.path),
+              fit: BoxFit.cover,
+            ),
+          ),
+          child: Padding(
+            padding:
+                const EdgeInsets.symmetric(horizontal: Dimensions.paddingLarge),
+            child: Center(
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Column(
+                      mainAxisAlignment: MainAxisAlignment.spaceAround,
+                      children: [
+                        _nimbleLogo,
+                        const SizedBox(height: 24.0),
+                        _forgotPasswordInstruction(context),
+                        const SizedBox(height: 100.0),
```

Comment:
Please check again cause in the design Figma, I see the padding as `96.0` instead of `100.0`
```suggestion
                        const SizedBox(height: 96.0),
```

<img width="300" alt="Screenshot 2023-04-12 at 2 21 24 PM" src="https://user-images.githubusercontent.com/25218255/231382562-36a7a6a8-7140-4960-ac11-912625225349.png">